components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabsAdapter.kt

```diff
@@ -56,6 +56,8 @@ class TabsAdapter(
 
     override fun updateTabs(tabs: Tabs) {
         this.tabs = tabs
+
+        tabsTray.scrollToPosition(tabs.selectedIndex)
```

Comment:
Do we want to scroll to position every time we call `updateTabs`? We update the adapter if we delete a tab

Reply:
Fair point! I see now that it can be more jarring than I anticipated, I can move this to happen for every other case instead.